Falcon-7B/Dockerfile

```diff
@@ -2,11 +2,12 @@ from ubuntu:latest
 
 RUN apt update && apt install -y python3 python3-pip
 
-RUN pip install transformers torch einops xformers accelerate
-
+RUN pip install transformers torch einops xformers accelerate falcon
+RUN pip install --upgrade transformers
```

Comment:
I don't think you need to do this one, transformers is already being installed above?

Reply:
There used to be an error in earlier testing versions when it was not upgraded, this line remained and is indeed unnecessary, thank you!